tickit/devices/cryostream/base.py

```diff
@@ -80,6 +82,9 @@ async def ramp(self, ramp_rate: int, target_temp: int) -> None:
         ):
             self.gas_flow = 10
         else:
+            # NOTE: It does not seem like this code can be reached.
+            # It is not possible to set the target temperature above 90000ck with
+            # this function.
             self.gas_flow = 5
```

Comment:
Should this code be removed? Or should conditionals be altered such that it is reachable?

Reply:
Depends where this bit of logic comes from, if this is how the device works then I don't see how it could ever have a gas flow rate other than 10. I suppose setting `max_temp` to above 90000 would allow this code to be accessed but I don't know whether this is intended.

Reply:
Given that's 627°c it seems a bit off for a nominally sub-ambient device. Likely warrants further investigation into the device spec.

Reply:
Sounds sensible, I would be very interested in seeing where the logic for the devices came from. It would be very useful for improving the tests.

Reply:
According to the Cryostream 800 documentation (https://confluence.diamond.ac.uk/display/I041Tech/%2820150923%29+I04-1+purchase+wishlist+for+2016?preview=/39486789/40305915/Cryostream%20800.pdf) the gas flow is intended to be higher: "[a]t temperatures < 90 Kelvin". Therefore we can confidently hypothesize that there is a trailing '0' on '90000' in this conditional, the removal of which would fix this problem.